src/css/style.scss

```diff
@@ -54,34 +48,40 @@ h2 {
   height: 100%;
 }
 
-.sidebar-header-name {
-  text-align: center;
-}
+  // .sidebar-header-name {
+  //   text-align: center;
+  // }
+// }
 
-.sidebar-header-userInfo {
+/*!! This is being used in 9 different places in page-manip.
+Why is it the header for 'Friends'?
+Come back to this !!*/
+.sidebar-friendslist-header {
```

Comment:
Personally, I still find a class-name like this a bit confusing. In my head headers and sidebars are different things. Would this be useable if it was just `friendsList`? Especially given that it's being used in so many places